packages/expo-updates/android/src/main/java/expo/modules/updates/UpdatesConfiguration.kt

```diff
@@ -47,7 +47,7 @@ class UpdatesConfiguration private constructor (
     ),
     updateUrl = overrideMap?.readValueCheckingType<Uri>(UPDATES_CONFIGURATION_UPDATE_URL_KEY) ?: context?.getMetadataValue<String>("expo.modules.updates.EXPO_UPDATE_URL")?.let { Uri.parse(it) },
     sdkVersion = overrideMap?.readValueCheckingType<String>(UPDATES_CONFIGURATION_SDK_VERSION_KEY) ?: context?.getMetadataValue("expo.modules.updates.EXPO_SDK_VERSION"),
-    runtimeVersion = overrideMap?.readValueCheckingType<String>(UPDATES_CONFIGURATION_RUNTIME_VERSION_KEY) ?: context?.getMetadataValue<Any>("expo.modules.updates.EXPO_RUNTIME_VERSION")?.toString()?.replaceFirst("^string:".toRegex(), ""),
+    runtimeVersion = overrideMap?.readValueCheckingType<String>(UPDATES_CONFIGURATION_RUNTIME_VERSION_KEY) ?: UpdatesConfiguration.getRuntimeVersion(context),
```

Comment:
Can you confirm that this works as expected and preserves the multi digit format?

Reply:
I've tested this locally in an app with the appVersion update policy and version = 1.10, and the value is preserved correctly as expected.

The new code is designed to look for the string resource first, and fall back to looking for the metadata value if the string resource is not found (so that older manifests still work). I'll be adding some Android tests for these.